### DISCUSSION TITLE
GraphQL dependencyGraphManifests has no way to filter on a specific path or packagemanager

### DISCUSSION THREAD

**Discussion by fproulx-boostsecurity** [community/community#47807] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Hi,

I am using GraphQL preview `application/vnd.github.hawkgirl-preview+json` and I have a GraphQL query like:
```
query {
	repositoryOwner(login:"snip-snip") {
		repositories(first:10 isFork:false privacy:PRIVATE orderBy:{field:NAME direction:ASC}) {
			edges {
				node {
					nameWithOwner
					dependencyGraphManifests(first:10) {
						edges {
							node {
								filename
								dependenciesCount
								dependencies(first:10) {
									 edges {
										node {
											hasDependencies
											packageManager
											packageName
											requirements
										}
									}
								}
							}
						}
					}
				}
			}
		}
	}
}
```

I wish I could restrict my search to depencygraph for a specific ecosystem / packagemanager (first instance just NPM or GitHubAction or PyPI), because for instance I may have a lot of PyPI stuff in my graph and I may want to focus on just digging into NPM first.

**Comment by semihural-tomtom** [comment]
@fproulx-boostsecurity did you able to find any workaround? I am looking similar thing, just filter actions.
